tests/Unit/PointwiseFunctions/GeneralRelativity/Test_KerrSchildCoords.cpp

```diff
@@ -121,10 +126,33 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.GeneralRelativity.KerrSchildCoords",
   test_coord_transformation_on_xy_plane(d);
   test_coord_transformation_on_xy_plane(dv);
 
-  gr::KerrSchildCoords kerr_schild_coords(0.8624);
+  gr::KerrSchildCoords kerr_schild_coords(0.8624, 0.151);
   test_serialization(kerr_schild_coords);
 
-  gr::KerrSchildCoords kerr_schild_coords_copy(0.8624);
+  gr::KerrSchildCoords kerr_schild_coords_copy(0.8624, 0.151);
   test_move_semantics(std::move(kerr_schild_coords),
                       kerr_schild_coords_copy);  // NOLINT
 }
+
+// [[OutputRegex, The mass must be positive]]
+[[noreturn]] SPECTRE_TEST_CASE(
+    "Unit.PointwiseFunctions.GeneralRelativity.KerrSchildCoords.Mass",
+    "[Unit][PointwiseFunctions]") {
+  ASSERTION_TEST();
+#ifdef SPECTRE_DEBUG
+  gr::KerrSchildCoords ks_coords(-4.21, 0.999);
+  ERROR("Failed to trigger ASSERT in an assertion test");
+#endif
+}
+
+// [[OutputRegex, The dimensionless spin must be in the range \(-1, 1\)]]
+[[noreturn]] SPECTRE_TEST_CASE(
+    "Unit.PointwiseFunctions.GeneralRelativity.KerrSchildCoords.DimlessSpin",
+    "[Unit][PointwiseFunctions]") {
+  ASSERTION_TEST();
+#ifdef SPECTRE_DEBUG
+  gr::KerrSchildCoords ks_coords(0.15, -1.3);
+  gr::KerrSchildCoords other_ks_coords(1.532, 4.2);
```

Comment:
this line doesn't test anything as the previous one will trigger the Assert, so please remove it or separate it into another test.